src/types.ts

```diff
@@ -1,140 +1,343 @@
-import { Asset, Transaction } from './gnosis-safe/types';
-import { Proposal, TokenMetaData, Vote, zDAO } from './snapshot-io/types';
+import {
```

Comment:
This file has grown too big (to the point you have to use code headers)

Split this out into a `types/` folder which exports all these and create multiple files for the implementations; ie `primitives` `config` `enumerations` etc.